src/shells/bash.rs

```diff
@@ -19,7 +19,7 @@ __pazi_add_dir() {
 if [[ -z "${PROMPT_COMMAND}" ]]; then
     PROMPT_COMMAND="__pazi_add_dir;"
 else
-    PROMPT_COMMAND="$(read newVal <<<"$PROMPT_COMMAND"; echo "${newVal%;}; __pazi_add_dir;")"
+    PROMPT_COMMAND="$(read -r newVal <<<"$PROMPT_COMMAND"; echo "${newVal%;}; __pazi_add_dir;")"
```

Comment:
I think I'd rather switch to:

```
PROMPT_COMMAND="__pazi_add_dir;\$PROMPT_COMMAND"
```

And cite https://github.com/clvv/fasd/blob/90b531a5daaa545c74c7d98974b54cbdb92659fc/fasd#L132-L136 as inspiration.

The stack overflow comment claiming this was safe is clearly wrong, and I'd trust fasd's method to have been more battle-tested by now.